jep/211/README.adoc

```diff
@@ -350,6 +350,23 @@ Non-goals in the specification are defined to limit the scope of work.
 The main objective is to get Jenkins running with Java 11,
 there will be follow-up tasks to cleanup Illegal Reflective Access warnings and to adopt new features.
 
+=== Support of Java 10 and 12
+
+Originally this JEP was targeting support of Java 10 and Java 11 early access.
+Java 10 is in the End of Life starting from 25.09.2018 when Java 11 was officially released.
+To limit the scope, Java 10 support was excluded from this JEP.
+We will not be testing new patches with Java 10 and providing Docker packages with this version.
+It may be possible to run Jenkins WAR files with Java 10 as it described link:https://jenkins.io/blog/2018/06/17/running-jenkins-with-java10-11/[here],
```

Comment:
NIT: as it described -> as it is described?? I don't understand well enough

Reply:
```suggestion
It may be possible to run Jenkins WAR files with Java 10 as it is described link:https://jenkins.io/blog/2018/06/17/running-jenkins-with-java10-11/[here],
```